sound/soc/sof/sof-pci-dev.c

```diff
@@ -517,6 +517,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_ALDERLAKE)
 	{ PCI_DEVICE(0x8086, 0x7ad0),
 		.driver_data = (unsigned long)&adls_desc},
+	{ PCI_DEVICE(0x8086, 0x51c8),
+		.driver_data = (unsigned long)&tgl_desc},
```

Comment:
just to make sure this isn't a copy-paste error? ADL-P does indeed have to use `tgl_desc`?

Reply:
Ack @lyakh , that is intentional.